### PR TITLE
Add vote Nov 15 2022

### PR DESCRIPTION
**Pull request by arwer13**
1. send DAI 1,500,000 to Pool Maintenance Labs Ltd. 0x17F6b2C738a63a8D3A113a228cfd0b373244633D
2. send LDO 220,000 to Pool Maintenance Labs Ltd. 0x17F6b2C738a63a8D3A113a228cfd0b373244633D
3. send  DAI 500,000 to Argo Technology Consulting Ltd. 0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956
4. send DAI 250,000 to RCC 0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437
5. send LDO 177,726 to RCC 0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437

### CODE REVIEW

File: tests/test_2022_11_15.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+)
+from utils.finance import ZERO_ADDRESS
+from brownie.network.transaction import TransactionReceipt
+from brownie import interface
+
+
+dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+lido_dao_token = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+weth_token = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+dai_token_address = "0x6B175474E89094C44Da98b954EedeAC495271d0F"
```
Comment (mymphe) [inline tests/test_2022_11_15.py:18]: I think these addresses are present in `utils/mainnet_config.py`. I'd suggest using that file as the single source of truth.

Comment (TheDZhon) [reply tests/test_2022_11_15.py:18]: I would rather hardcoded them in tests indeed (because config can be changed remain unnoticed)

Comment (mymphe) [reply tests/test_2022_11_15.py:18]: Okay, yeah, I see where you are coming from :) But if we don't trust the global config file, that kinda defeats the whole purpose of having one.

Comment (TheDZhon) [reply tests/test_2022_11_15.py:18]: The global config is a good thing when you write scripts and utils. Though, everything should have a fuse (especially, if we are transferring funds). That's why I propose to have the hardcoded test constants (i.e., break the circuit if something is wrong).

File: tests/test_2022_11_15.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    validate_token_payout_event,
+)
+from utils.finance import ZERO_ADDRESS
+from brownie.network.transaction import TransactionReceipt
+from brownie import interface
+
+
+dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+lido_dao_token = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+weth_token = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
```
Comment (zuzueeka) [inline tests/test_2022_11_15.py:17]: It seems we don't need weth_token

Comment (arwer13) [reply tests/test_2022_11_15.py]: fixed, thanks!

File: tests/test_2022_11_15.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        - pool_maintenance_labs_dai_payout.amount
+        - argo_technology_consulting_dai_payout.amount
+        - rcc_dai_payout.amount
+    ), "DAI balance of DAO Agent must decrease by the correct amount"
+
+    # Check LDO payment
+    assert (
+        ldo_token.balanceOf(pool_maintenance_labs_address)
+        == pool_maintenance_labs_ldo_before + pool_maintenance_labs_ldo_payout.amount
+    ), "LDO balance of RCC multisig must increase bye the correct amount"
```
Comment (zuzueeka) [inline tests/test_2022_11_15.py:118]: typo in "bye". And one more below

Comment (arwer13) [reply tests/test_2022_11_15.py]: fixed

**Review by zuzueeka — APPROVED** [review]
lgtm

**Pull request merged**